src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -79,9 +79,17 @@ public static class PrereleaseCompatibilityTransformer
                         case "http://json.schemastore.org/sarif-2.1.0-rtm.1":
                         case "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.1.json":
                         {
-                            modifiedLog |= ApplyRtm2and3Changes(logObject);
```

Comment:
>ApplyRtm2and3Changes [](start = 43, length = 20)

There were no transformer changes for 2 and 3, this helper just updated the schema. I've made this an explicitly named operation.